windows/src/engine/kmcomapi/com/keyboardlanguages/keymankeyboardlanguagesinstalled.pas

```diff
@@ -94,8 +94,8 @@ procedure TKeymanKeyboardLanguagesInstalled.DoRefresh;
     FProfiles := TStringList.Create;
     with TRegistryErrorControlled.Create do
     try
-      RootKey := HKEY_LOCAL_MACHINE;
-      RootPath := GetRegistryKeyboardInstallKey(FOwner.ID) + SRegKey_LanguageProfiles;
+      RootKey := HKEY_LOCAL_MACHINE;  /// TODO: Is this correct? Do we ever have any
```

Comment:
Intentionally-left `TODO`?

Reply:
Good question. I think I'd like to add at the end of the line: `// TODO: finish writing this todo`